src/test/java/com/wikia/webdriver/common/dataprovider/MercurySEODataProvider.java

```diff
@@ -0,0 +1,90 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import com.wikia.webdriver.common.core.url.Page;
+import org.testng.annotations.DataProvider;
+
+public class MercurySEODataProvider {
+
+    @DataProvider
+    public static Object[][] htmlTitles() {
+        return new Object[][] {
+            {
+                new Page("sktest123", "/wiki/Sktest123_Wiki"), "Sktest123 Wiki - Wikia"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sktest123" 27 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)